lib/fast_jsonapi/multi_to_json.rb

```diff
@@ -76,23 +87,9 @@ def self.to_json_method
       encode_method << "\nend"
     end
 
-    def self.to_json(object)
-      _fast_to_json(object)
-    rescue NameError
-      define_to_json(FastJsonapi::MultiToJson)
-      _fast_to_json(object)
-    end
-
-    def self.define_to_json(receiver)
-      cl = caller_locations[0]
-      method_body = to_json_method
-      logger.debug { "Defining #{receiver}._fast_to_json as #{method_body.inspect}" }
-      receiver.instance_eval method_body, cl.absolute_path, cl.lineno
-    end
-
-    def self.reset_to_json!
-      undef :_fast_to_json if method_defined?(:_fast_to_json)
-      logger.debug { "Undefining #{receiver}._fast_to_json" }
```

Comment:
❓ It's not used in the normal lifecycle of the app and because there's no test in the suite, but I wouldn't say it serves no purpose.  Especially if this module were made `module_function` and included.

I do have some tests I started writing in https://github.com/bf4/fast_multi_json but they didn't cover this specifically